src/components/PopupWithConfirmation.js

```diff
@@ -0,0 +1,22 @@
+import Popup from './Popup.js';
+
+export default class PopupWithConfirmation extends Popup {
+    constructor(popupSelector) {
+        super({ popupSelector });
+        this._form = this._popupElement.querySelector('.modal__form');
```

Comment:
When I open the confirmation popup the submit button is disabled

Reply:
I've updated the confirmation popup element so its button has a click event listener instead of a form submission event listener. That should circumvent the form validation issue on a popup with no form inputs. Thank you.